tools/clang/tools/dxcvalidator/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ set( LLVM_LINK_COMPONENTS
   DXIL
   DxilContainer
   DxilHash
+  DxilRootSignature
   DxilValidation
+  HLSL       # for AreDxilResourcesDense and DxilSignatureAllocator which are used in DxilValidation
```

Comment:
Given that there are no code changes in this directory, I don't know why any changes would be needed here.

Reply:
Moved to CMakeLists.txt for dxildll.

Reply:
But gcc complains after remove change here.

Reply:
Okay. I suspect I know what's going on here. I think that these dependencies were always there, but they were satisfied indirectly by other libraries by chance. The linux linker is really particular that way.